worlds/witness/items.py

```diff
@@ -176,9 +176,16 @@ def get_filler_items(self, quantity: int) -> Dict[str, int]:
 
         # Read trap configuration data.
         trap_weight = self._world.options.trap_percentage / 100
-        filler_weight = 1 - trap_weight
+        trap_items = {name: self._world.options.trap_weights.get(name, 0)
```

Comment:
This can be removed; since the keys are non-optional, there's no reason to have a default.
```suggestion
        trap_items = {name: self._world.options.trap_weights.get(name)
```